src/api/app/controllers/webui/packages/meta_controller.rb

```diff
@@ -12,9 +12,12 @@ class MetaController < Webui::WebuiController
       after_action :verify_authorized, only: :update
 
       def show
-        return if @project.scmsync.present?
-
-        @meta = @package.render_xml
+        if @project.scmsync.present?
+          flash.now[:error] = "Error while retrieving the Meta file: package not found."
```

Comment:
This will be *very* confusing for people as other parts of the UI work with scmsync packages.

Reply:
https://github.com/openSUSE/open-build-service/blob/545e6c90210708322055c9aa197823cac2c8d3f3/src/api/app/controllers/webui/package_controller.rb#L39-L44

Reply:
Excellent